Dockerfile

```diff
@@ -45,6 +53,8 @@ COPY lib/pandaseq /app/lib/pandaseq
 COPY lib/vsearch /app/lib/vsearch
 COPY lib/casper /app/lib/casper
 COPY lib/swarm /app/lib/swarm
+COPY lib/mistag_filter /app/lib/mistag_filter
+COPY lib/abundance_filters /app/lib/abundance_filters
```

Comment:
Because I did not yet grasp the full scope of GitHub as a tool for crunching changes one bit at a time and therefore my commit contained another bit corresponding to a further change...